webpack.config.js

```diff
@@ -0,0 +1,45 @@
+const path = require('path')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+
+module.exports = {
+  entry: {
+    app: ['./src/index.js']
+  },
+  output: {
+    filename: 'bundle.js',
+    path: path.join(__dirname, 'dist')
+  },
+  // Enable sourcemaps for debugging webpack's output.
+  devtool: 'source-map',
```

Comment:
[https://webpack.js.org/configuration/devtool/](url)

this is for production, we can use cheap-eval-source-map